test/integration/net-driver.bats

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bats
+
+function setup {
+
+  # run plugin
+  nohup docker run --privileged --net=host --rm -v /run/docker/plugins:/run/docker/plugins routed-plugin --gateway 10.100.0.1 --mtu 9000 --debug > $BATS_TMPDIR/routed_plugin_test.out 2>&1 &
```

Comment:
docker run with `-d` should start it dettached in background mode and return the container id.

you can provide a name, for example, `--name routed-network-plugin` and then you can shutdown with `docker kill routed-network-plugin`